show/main.py

```diff
@@ -850,6 +851,16 @@ def breakout(ctx):
             platformDict[port_name].update(hwskuDict[port_name])
             platformDict[port_name].update(curBrkout_tbl[port_name])
 
+            # Update portname of child ports
+            offset = int(port_name.split(PORT_STR)[1])
+            total_lane = len(platformDict[port_name]["lanes"].split(','))
+            child_ports = ",".join(PORT_STR + str(offset+lane) for lane in range(total_lane))
```

Comment:
How is the current breakout mode accounted while generating child_port names?

Reply:
I mean, 
current mode.   child ports
==========   ========
1x100G              "Ethernet0"
4x25G               "Ethernet0,Ethernet1,Ethernet2,Ethernet3"
2x50G               "Ethernet0,Ethernet2"

Reply:
Thats a good idea. Modified the code as requested.